iree/test/e2e/linalg_ext_ops/scan.mlir

```diff
@@ -2,65 +2,80 @@ func @scan_1d_dim0_inclusive_sum() {
   %input = util.unfoldable_constant dense<[1.0, 2.0, 3.0, 4.0, 5.0, 6.0]> : tensor<6xf32>
 
   %init = linalg.init_tensor [6] : tensor<6xf32>
-  %c0 = arith.constant 0.0 : f32
-  %0 = iree_linalg_ext.scan
+  %t0 = util.unfoldable_constant dense<0.0> : tensor<f32>
```

Comment:
Maybe have a non-zero value here so that you can truly check that the initial value has been added.

Reply:
Sure will change.